mm2src/mm2_main/tests/docker_tests/mod.rs

```diff
@@ -12,4 +12,4 @@ mod swaps_file_lock_tests;
 
 // dummy test helping IDE to recognize this as test module
 #[test]
-fn dummy() { assert!(true) }
+fn dummy() {}
```

Comment:
I don't use IDEs, can anyone test this?

cc @caglaryucekaya @shamardy @borngraced @laruh

Reply:
Works fine. Removed the test and the IDE didn't recognize `docker_tests` as a test mod
https://github.com/KomodoPlatform/atomicDEX-API/blob/75663fff7f3f052b0b153d2695e7b8b84480513f/mm2src/mm2_main/tests/docker_tests_main.rs#L29
With the change in this PR it's still recognized as a test mod.

Reply:
It was left because clippy didn't recognize it as problem. I think that is outdated workaround, I've got rid of it. Thank you @laruh